demonstrations/tutorial_how_to_use_registers.metadata.json

```
@@ -0,0 +1,42 @@
+{
+    "title": "How to use wire registers",
+    "authors": [
+        {
+            "username": "austingmhuang"
+        }
+    ],
+    "dateOfPublication": "2024-07-23T00:00:00+00:00",
+    "dateOfLastModification": "2024-07-23T00:00:00+00:00",
```

```suggestion
    "dateOfPublication": "2024-09-05T00:00:00+00:00",
    "dateOfLastModification": "2024-09-05T00:00:00+00:00",
```

I think this is correct.